### DISCUSSION TITLE
Image Viewer auto-hide toolbar

### DISCUSSION THREAD

**Discussion by NjlsShade** [linuxdeepin/developer-center#3198] [Features Request & Ideas | 特性请求 & 头脑风暴]
### SUMMARY | 问题概要

The toolbar in Image Viewer can sometimes obscure the bottom of images you're trying to view, the toolbar should be hideable by clicking the image content (Similar to other photo viewers like the one in Windows 11) or automatically when the mouse does not move over the window for some time.

### OBSERVED RESULT | 观察到的结果

Text at the bottom of this image is not visible:
![Screenshot_deepin-image-viewer_20220528161601](https://user-images.githubusercontent.com/15788214/170841667-98494f50-e02e-401c-ad01-b821196c0c80.png)

Source image:
![Image Viewer test](https://user-images.githubusercontent.com/15788214/170841682-c9b360c2-a451-47b2-9593-34188f4becb3.png)

### SOFTWARE/OS VERSIONS | 软件/系统版本信息

Using the latest deepin 20.5. Image Viewer 5.8.13.

**Comment by BLumia** [comment]
@meiyixiang : 需求是希望工具栏可以自动隐藏（比如当点击图片的时候，或者鼠标几秒没有移动的时候），以免遮挡图片，影响图像的查看。

**Comment by meiyixiang** [comment]
Thank you for your suggestions. I have recorded and fed back to the product, and hope to support it as soon as possible

**Comment by BLumia** [comment]
This is probably get fixed in v20.6 release with `deepin-image-viewer 5.9.1-1`, could you check if the current behavior can fit your need? Thanks!

**Comment by meiyixiang** [comment]
Version 20.6 supports hiding but only hides toolbars when you need to slide the scroll wheel
